packages/trace-viewer/vite.config.ts

```
@@ -30,6 +30,7 @@ export default defineConfig({
   ],
   define: {
     'process.env': {},
+    '__APP_VERSION__': JSON.stringify(process.env.npm_package_version),
```

Does this silently fail if the build is not run from within `npm` scripts?